src/server/services/CheckRole.js

```diff
@@ -23,3 +23,18 @@ module.exports.isAdmin = function(rawRoles) {
   }
   return true;
 };
+
+module.exports.isInstructor = function(rawRoles) {
+  const roles = rawRoles.map(role =>
+    role.substr(role.lastIndexOf('#') + 1, role.length).toLowerCase()
+  );
+  if (
+    !roles.includes('instructor') &&
+    !roles.includes('teacher') &&
+    !roles.includes('administrator') &&
+    !roles.includes('admin')
```

Comment:
@maxpluez The isAdmin() function can be called here instead, right?

Reply:
hmm isn't it quicker to do direct boolean check then calling function?

Reply:
Hmm, I was thinking more along the lines that calling isAdmin() here would mean one fewer thing to update if admin checking ever changes in the future